lib/matplotlib/figure.py

```diff
@@ -682,11 +682,10 @@ def set_size_inches(self, *args, **kwargs):
         matplotlib.Figure.get_size_inches
         """
 
-        forward = kwargs.get('forward', False)
-        if len(args) == 1:
-            w, h = args[0]
-        else:
-            w, h = args
+        # the width and height have been passed in as a tuple to the first
```

Comment:
@efiring Yup I agree internal code so not much point here, but what about on the lines above in the function definition?  See ``FigureManagerTkAgg.resize`` for the comment I referred to, should we do something similar here?  I have also just done a search for sfw and sfh and only these methods come up.

@jhamrick I think ``set_size`` sounds confusing as it contains no units, it inches, centimetres, pixels, etcetera.  I guess we do it by inches because of dpi (dots per square inch), a rather odd unit, but used everywhere in graphics, I have never seen dpcm :(.  Perhaps you should also take a look at